contracts/src/v0.8/tests/ChainReaderTestContract.sol

```diff
@@ -0,0 +1,103 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8;
+
+struct TestStruct {
```

Comment:
We no longer allow new files in the generic folders (e.g. tests, interfaces, ...). All files should be in a project folder like Automation, VRF, .... 

I see you added a new compile file, that indicates this is a new project?

Reply:
In that case the `shared` folder probably fits best

Reply:
Looks like the place @RensR is recommending is here:
https://github.com/smartcontractkit/chainlink/tree/develop/contracts/src/v0.8/shared/test

We can move it there, but I'm curious what the difference is between `tests` and `shared/test`.  Why do we have two different places where there are product-independent tests?  Any plans to combine these at some point?

Reply:
Thanks for doing this, I'll resolve the comment so it's clear that's done :).